src/main/java/persistence/sql/dml/QueryBuilder.java

```diff
@@ -0,0 +1,112 @@
+package persistence.sql.dml;
+
+import jakarta.persistence.Column;
+import jakarta.persistence.GeneratedValue;
+import jakarta.persistence.Table;
+import jakarta.persistence.Transient;
+
+import java.lang.reflect.Field;
+import java.lang.reflect.InvocationTargetException;
+import java.sql.ResultSet;
+import java.sql.SQLException;
+import java.util.ArrayList;
+import java.util.Arrays;
+import java.util.List;
+import java.util.stream.Collectors;
+
+public class QueryBuilder implements Database{
+
+    private static final String INSERT_QUERY_FORMAT = "INSERT INTO %s (%s) VALUES (%s)";
+    private static final String SELECT_QUERY_FORMAT = "SELECT * FROM %s";
+    Class<?> clazz;
+    public QueryBuilder(Class<?> clazz) {
+        this.clazz = clazz;
+    }
+
+    @Override
+    public ResultSet executeQuery(String sql) {
+
+        try {
+
+        } catch (Exception e) {
+
+        }
+         return null;
+    }
+
+    public List<Object> findAll() {
+        String sql = String.format(SELECT_QUERY_FORMAT, getTableName());
+        ResultSet rs = executeQuery(sql);
+        List<Object> results = new ArrayList<>();
+
+        try {
+            while(rs.next()) {
+                Object instance = clazz.getDeclaredConstructor().newInstance();
+
+                for(Field field : clazz.getDeclaredFields()) {
+                    field.setAccessible(true);
+
+                    if(!field.isAnnotationPresent(Transient.class)) {
+                        Object value = rs.getObject(field.getName());
+                        field.set(instance, value);
+                    }
+                }
+                results.add(instance);
+            }
+        } catch (Exception e) {
+            e.printStackTrace();
+        }
+
+        return results;
+    }
+    public String insert() {
```

Comment:
insert query도 아래와 같이 구현해보시면 좋을 것 같습니다!
아래와 같이 구현하면 어떤 장점을 얻을 수 있을까요? 

```suggestion
    public String insert(Object entity) {
```